CMakeLists.txt

```diff
@@ -48,6 +49,7 @@ set(THIRDPARTY_SOURCES_DIR "${CMAKE_CURRENT_SOURCE_DIR}/3rdparty" CACHE PATH
 
 ## Dependencies
 find_package(Poco REQUIRED COMPONENTS Net Foundation)
+find_package(console_bridge REQUIRED)
```

Comment:
this should be a transitive dependency coming from Pinocchio library. So we don't need to add that here. We had this issue that libfranka couldn't find the console_bridge. Try to remove ros2 packages including the console_bridge that solves the issue e.g. ros_${distro}_pinocchio or ros_${distro}_console_bridge